pysipp/launch.py

```diff
@@ -56,8 +56,7 @@ async def run(
         for cmd in cmds:
             log.debug(
                 "launching cmd:\n\"{}\"\n".format(cmd))
-            # proc = await trio.open_process(
-            proc = trio.Process(
+            proc = await trio.open_process(
```

Comment:
do we want the async context manager interface here?

I'm pretty sure there's [some teardown](https://github.com/python-trio/trio/blob/master/trio/_subprocess.py#L183) we may find useful? 

I'm not sure how this is going to change though given python-trio/trio#1568.

Reply:
aren't changes in this trio PR going to make it so you can use `trio.open_process` inside a nursery?
in pysipp we want the Scenario API to not be async right?

Reply:
> do we want the async context manager interface here?

what do you have in mind? isn't the teardown part of finalize that will be called from sync code?

Reply:
> in pysipp we want the Scenario API to not be async right?

Right, I think that's what we have to aim for at the outset.

> isn't the teardown part of finalize that will be called from sync code?

Yes you're right, so I guess I was thinking we should trigger the shutdown code I linked to, but because it's getting called from sync code we might have to either call `Process.aclose()` explicitly or we use an async exit stack. I'm thinking the former is simpler and cleaner?